src/codegen/mod.rs

```diff
@@ -3478,6 +3478,7 @@ impl CodeGenerator for ObjCInterface {
 }
 
 pub fn codegen(context: &mut BindgenContext) -> Vec<P<ast::Item>> {
+    let _t = context.timer("codegen");
```

Comment:
This is going to make codegen's timer include all of our analyses that get run from `gen`, which we don't really want here.

Let's move the timer into the closure passed to `gen` below.